llpc/include/llpc.h

```diff
@@ -131,6 +131,7 @@ enum CacheAccessInfo : uint8_t {
 /// Represents output of building a graphics pipeline.
 struct GraphicsPipelineBuildOut {
   BinaryData pipelineBin;              ///< Output pipeline binary data
+  void *palMetaDta;                    ///< Pal meta data.
```

Comment:
This isn't PAL metadata, it's the `FragmentOutputs` metadata. Let's call it `fsOutputMetaData`. And same for the parameter of `BuildColorExportShader`.